src/index.tsx

```diff
@@ -157,9 +148,42 @@ var saveToWebmarkFolder = (url: string | undefined, title: string | undefined):
     );
 }
 
-var getRandomUrlFromFolder = (): string => {
-    //TODO: implement getRandomUrlFromFolder
-    return 'https://example.com/random';
+let getRandomUrlFromFolder = (): void => {
+    chrome.storage.sync.get(
+        ['webmarkFolderId'],
+        (result?) =>{
+            console.log('webmarkFolderId found.');
+            let webmarkFolderId: string = result!['webmarkFolderId'];
+            chrome.bookmarks.getSubTree(
+                webmarkFolderId,
+                (bookmarkTreeNodes) =>{
+                    let urlList : Array<string> = [];
+                    for (let node of bookmarkTreeNodes) {
+                        if(node.children != undefined){
```

Comment:
Is `!=` (as opposed to `!==`) intentional here?

Reply:
not really, will update